src/services/audius-api-client/AudiusAPIClient.ts

```diff
@@ -1142,6 +1169,8 @@ class AudiusAPIClient {
   }
 }
 
-const instance = new AudiusAPIClient()
+const instance = new AudiusAPIClient({
+  overrideEndpoint: 'https://discovery2-prod.sandbox.audius.co/'
```

Comment:
remove!

Reply:
I knew I was gonna leave this in haha, good catch